my_code.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import matplotlib
```

Comment:
from matplotlib import pyplot should have been better, since it is used directly as pyplot below